views/windows.tmpl

```diff
@@ -3,7 +3,7 @@
 <html lang="en">
 
 {{template "html-head" "Decred Windows List"}}
-<body class="{{ theme }}" data-controller="ticketwindow">
+<body class="{{ theme }}" data-controller="ticketwindow blocklist">
     {{template "navbar" . }}
     <div class="container main">
```

Comment:
The `data-controller="ticketwindow blocklist"` bit seems to be an attribute of the `div` rather than `body` in other templates.  Is this correct and OK?